### ISSUE TITLE
Support for Mapgeo Version 17

### ISSUE THREAD

**Issue by TheKillerey** [tarngaina/lol_maya#10]
![image](https://github.com/tarngaina/lol_maya/assets/18242778/fb1fafac-d48e-4764-8568-2c5537845d1a)

**Comment by JunioCesar96** [comment]
Did the creator abandon the project?

**Comment by MelancholySlime** [comment]
> Did the creator abandon the project?

Yes and he moving all to Lemon3d from LtMao instead. So if u need newest plugin then use download Ltmao project and install Lemon3D
